kubectl-metalk8s/pkg/salt/client.go

```diff
@@ -62,6 +70,51 @@ func NewClient(creds *Credential, caCertData []byte) (*Client, error) {
 	}, nil
 }
 
+// Create a new Salt API client from a given config (basically a kubeconfig)
+func NewForConfig(config *rest.Config) (*Client, error) {
```

Comment:
`NewFromConfig` I guess.

Reply:
Just took the same name as the kubernetes one it's "new client for this config"